setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
+    description='Python client for the Yola API',
+    author='Yola',
+    author_email='engineers@yola.com',
+    license='MIT (Expat)',
+    url='https://github.com/yola/yolapy',
+    packages=find_packages(),
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2',
```

Comment:
I'm a dirty lying rat bastard. `demands`' tests don't work on Python 3, but you can `pip install demands` happily on Python 3.

```
$ pip install querylist
Collecting querylist
  Using cached querylist-0.2.0.tar.gz
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/var/folders/sb/2fk29xgs3rq4nkzdzxdcrhsw0000gn/T/pip-build-ycd6ck58/querylist/setup.py", line 5, in <module>
        from querylist import __version__, __author__
      File "/private/var/folders/sb/2fk29xgs3rq4nkzdzxdcrhsw0000gn/T/pip-build-ycd6ck58/querylist/querylist/__init__.py", line 3, in <module>
        from querylist import QueryList
    ImportError: cannot import name 'QueryList'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

      File "<string>", line 20, in <module>

      File "/private/var/folders/sb/2fk29xgs3rq4nkzdzxdcrhsw0000gn/T/pip-build-ycd6ck58/querylist/setup.py", line 5, in <module>

        from querylist import __version__, __author__

      File "/private/var/folders/sb/2fk29xgs3rq4nkzdzxdcrhsw0000gn/T/pip-build-ycd6ck58/querylist/querylist/__init__.py", line 3, in <module>

        from querylist import QueryList

    ImportError: cannot import name 'QueryList'

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/sb/2fk29xgs3rq4nkzdzxdcrhsw0000gn/T/pip-build-ycd6ck58/querylist
```